src/basis/basis.ts

```diff
@@ -90,8 +90,13 @@ export class Basis {
         return new Lattice().vectorArrays;
     }
 
+    static _roundValue(x: number) {
+        return math.precise(math.roundToZero(x));
```

Comment:
Duplicating the above in https://github.com/Exabyte-io/made.js/pull/109/files#diff-eeeeda24f289e664e12e92c6bd0225a49cbd6bae52a44d4137d1742e55d1ed85R44, should be in one place